tpm2/structures.go

```diff
@@ -119,12 +119,18 @@ func DecodePublic(buf []byte) (Public, error) {
 // precedence. ModulusRaw is used for key templates where the field named
 // "unique" must be a byte array of all zeroes.
 type RSAParams struct {
-	Symmetric  *SymScheme
-	Sign       *SigScheme
-	KeyBits    uint16
-	Exponent   uint32
-	ModulusRaw []byte
-	Modulus    *big.Int
+	Symmetric *SymScheme
+	Sign      *SigScheme
+	KeyBits   uint16
+	// The default Exponent (65537) has two representations; the
+	// 0 value, and the value 65537.
+	// If EncodeDefaultExponentAsZero is set, an exponent of 65537
```

Comment:
match the field name EncodeDefaultExponentAsZero -> encodeDefaultExponentAsZero

Reply:
Done.